web/packages/dmt-core/src/components/NewEntityButton.tsx

```diff
@@ -40,6 +40,7 @@ export function NewEntityButton(props: {
   )
   const [typeToCreate, setTypeToCreate] = useState<string>(type || '')
   const [loading, setLoading] = useState<boolean>(false)
+  // @ts-ignore
```

Comment:
the type error should be fixed in master branch, and not necessary to add the @ts-ignore